### DISCUSSION TITLE
SP API- PII Data

### DISCUSSION THREAD

**Discussion by mstr1989** [amzn/selling-partner-api-models#3695] [General]
Hi there, 

Despite completing our developer profile and receiving an email from Amazon confirmed they have authorised us to access it to fulfil our orders, we are still seeing that we are restricted. We requested an extension for this but we should have access and we are not sure why Amazon confirmed we are accepted yet we are not. Has anyone else had issues like this? 

Thanks, 
Mike

**Comment by MeenaAmz** [comment]
@mstr1989 Can you please explain what you mean by restricted? Are you not able to create applications in the seller or vendor central portal? We can try to help if you can provide more information.

**Reply by ghost** [reply]
I am also getting the same issue. I could not get PII information from getOrders SP API, could create reports of restricted type. Eventhough I have the respective role.Do I need to use different refresh token? or Is there any other alternative @MeenaAmz.

**Reply by MeenaAmz** [reply]
For accessing PII data, you need to have access to restricted roles. Please refer to the [roles guide](https://developer-docs.amazon.com/sp-api/docs/roles-in-the-selling-partner-api) to make sure you have access to the appropriate restricted role required to access the PII data.

**Reply by rugved1991** [reply]
@Arunkumar-ojc 
If you need to pull restricted information from the getOrders API operation, you would need to use the Restricted Data Token in the API call. Otherwise you can use your regular access token to call the restricted operations like getOrderBuyerInfo, etc.
For more information on how to use Restricted Data Token, please refer to this [use case guide](https://developer-docs.amazon.com/sp-api/docs/tokens-api-use-case-guide).

Best,
Rugved
Solutions Architect, SP-API

**Reply by ghost** [reply]
@MeenaAmz @rugved1991 Thank You.
